packages/medusa/src/api/admin/exchanges/[id]/inbound/shipping-method/[action_id]/route.ts

```diff
@@ -0,0 +1,110 @@
+import {
```

Comment:
Same question here, will not comment more on this anymore if there are other similar endpoints.